constraint/Makefile

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Nit: Same comments as on the go deps and Dockerfile apply here.

Reply:
Cleaned